libparsec/crates/platform_device_loader/Cargo.toml

```diff
@@ -13,6 +13,7 @@ workspace = true
 
 [features]
 test-with-testbed = ["libparsec_testbed"]
+vendored-keyring = ["keyring/vendored"]
```

Comment:
this is not keyring this is vendored, but libdbus. So this feature should be called `vendored-dbus`

Reply:
done